SUMMARY.md

```diff
@@ -1,58 +1,125 @@
-  * [A Principled Refactor](ch6.md#a-principled-refactor)
-  * [In Summary](ch6.md#in-summary)
-* [Chapter 7: Hindley-Milner and Me](ch7.md)
-  * [What's your type?](ch7.md#whats-your-type)
-  * [Tales from the cryptic](ch7.md#tales-from-the-cryptic)
-  * [Narrowing the possibility](ch7.md#narrowing-the-possibility)
-  * [Free as in theorem](ch7.md#free-as-in-theorem)
-  * [In Summary](ch7.md#in-summary)
-* [Chapter 8: Tupperware](ch8.md)
-  * [The Mighty Container](ch8.md#the-mighty-container)
-  * [My First Functor](ch8.md#my-first-functor)
-  * [Schrödinger’s Maybe](ch8.md#schrödingers-maybe)
-  * [Pure Error Handling](ch8.md#pure-error-handling)
-  * [Old McDonald had Effects…](ch8.md#old-mcdonald-had-effects)
-  * [Asynchronous Tasks](ch8.md#asynchronous-tasks)
-  * [A Spot of Theory](ch8.md#a-spot-of-theory)
-  * [In Summary](ch8.md#in-summary)
-* [Chapter 9: Monadic Onions](ch9.md)
-  * [Pointy Functor Factory](ch9.md#pointy-functor-factory)
-  * [Mixing Metaphors](ch9.md#mixing-metaphors)
-  * [My chain hits my chest](ch9.md#my-chain-hits-my-chest)
-  * [Theory](ch9.md#theory)
-  * [In Summary](ch9.md#in-summary)
+* [Chapter 01: What ever are we doing?](ch01.md)
+  * [Introductions](ch01.md#introductions)
+  * [A brief encounter](ch01.md#a-brief-encounter)
+* [Chapter 02: First Class Functions](ch02.md)
+  * [A quick review](ch02.md#a-quick-review)
+  * [Why favor first class?](ch02.md#why-favor-first-class)
+* [Chapter 03: Pure Happiness with Pure Functions](ch03.md)
+  * [Oh to be pure again](ch03.md#oh-to-be-pure-again)
+  * [Side effects may include...](ch03.md#side-effects-may-include)
+  * [8th grade math](ch03.md#8th-grade-math)
+  * [The case for purity](ch03.md#the-case-for-purity)
+  * [In Summary](ch03.md#in-summary)
+* [Chapter 04: Currying](ch04.md)
+  * [Can't live if livin' is without you](ch04.md#cant-live-if-livin-is-without-you)
+  * [More than a pun / Special sauce](ch04.md#more-than-a-pun--special-sauce)
+  * [In Summary](ch04.md#in-summary)
+  * [Exercises](ch04.md#exercises)
+* [Chapter 05: Coding by Composing](ch05.md)
+  * [Functional Husbandry](ch05.md#functional-husbandry)
+  * [Pointfree](ch05.md#pointfree)
+  * [Debugging](ch05.md#debugging)
+  * [Category Theory](ch05.md#category-theory)
+  * [In Summary](ch05.md#in-summary)
+  * [Exercises](ch05.md#exercises)
+* [Chapter 06: Example Application](ch06.md)
+  * [Declarative Coding](ch06.md#declarative-coding)
+  * [A flickr of functional programming](ch06.md#a-flickr-of-functional-programming)
+  * [A Principled Refactor](ch06.md#a-principled-refactor)
+  * [In Summary](ch06.md#in-summary)
+* [Chapter 07: Hindley-Milner and Me](ch07.md)
+  * [What's your type?](ch07.md#whats-your-type)
+  * [Tales from the cryptic](ch07.md#tales-from-the-cryptic)
+  * [Narrowing the possibility](ch07.md#narrowing-the-possibility)
+  * [Free as in theorem](ch07.md#free-as-in-theorem)
+  * [In Summary](ch07.md#in-summary)
```

Comment:
Above this line is a `  * [Constraints](ch07.md#constraints)` missing.

Reply:
Thanks, fixed with 4be90f94d52e6411a35b35841edc4389b2fc4a0f